chat-model/src/main/resources/canon/chat.json

```diff
@@ -1261,6 +1261,29 @@
           }
         }
       },
+      "ReactionDetailsPayload": {
+        "type": "object",
+        "extends": "#/components/schemas/MaestroPayload",
+        "properties": {
+          "action": {
+            "type": "string"
+          },
+          "reactionName": {
+            "type": "string"
+          },
+          "messageId": {
+            "type": "string"
+          },
+          "userId": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "timestamp": {
+            "type": "integer",
+            "format": "int64"
+          }
```

Comment:
Type should be 

{
            "$ref": "https://models.oss.symphony.com/core.json#/components/schemas/TimeStamp"
          }

Reply:
Done